testing/test/extensions/public/AzureMLKubernetes.Tests.ps1

```diff
@@ -58,11 +58,8 @@ Describe 'AzureML Kubernetes Testing' {
     It "Deletes the extension from the cluster with inference enabled" {
         # cleanup the relay and servicebus
         $relayResourceID = Get-ExtensionConfigurationSettings $extensionName $relayResourceIDKey
-        $serviceBusResourceID = Get-ExtensionConfigurationSettings $extensionName $serviceBusResourceIDKey
```

Comment:
Why is the service bus resource deletion being removed from the cleanup of the test case now?

Reply:
Because from now, service bus won't be created by default.